deployments/base/aero/deploy.ts

```diff
@@ -0,0 +1,94 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet, exp } from '../../../src/deploy';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const deployed = await deployContracts(deploymentManager, deploySpec);
+  return deployed;
+}
+
+async function deployContracts(
+  deploymentManager: DeploymentManager,
+  deploySpec: DeploySpec
+): Promise<Deployed> {
+  // AERO 
+  // AERO -> USD
+  const aeroUsdPriceFeed = await deploymentManager.deploy(
+    'AERO:priceFeed',
+    'pricefeeds/ScalingPriceFeed.sol',
+    [
+      '0x4EC5970fC728C5f65ba413992CD5fF6FD70fcfF0',
+      8
+    ]
+  );
+  // USDC
+  // USDC -> USD
+  const usdcToUsdPriceFeed = await deploymentManager.deploy(
+    'USDC:priceFeed',
+    'pricefeeds/ScalingPriceFeed.sol',
+    [
+      '0x7e860098F58bBFC8648a4311b374B1D669a2bc6B', // USDC -> USD
+      8,
+    ],
+    true
+  );
+  // WETH
+  // WETH -> USD
+  const ethToUsdPriceFeed = await deploymentManager.deploy(
+    'WETH:priceFeed',
+    'pricefeeds/ScalingPriceFeed.sol',
+    [
+      '0x71041dddad3595F9CEd3DcCFBe3D1F4b0a16Bb70', // ETH -> USD
+      8,                                            // decimals
+    ],
+    true
+  );
+  // wstETH
+  // wstETH -> USD
+  const wstETHToUsdPriceFeed = await deploymentManager.deploy(
```

Comment:
Should use wstETH:priceFeed from weth deployment since 1 stETH is close but not equal to 1 ETH

Reply:
This is made on purpose